docs/application/native/guides/multimedia/pose-detection.md

```diff
@@ -0,0 +1,343 @@
+
+   error_code = mv_create_engine_config(&imagedata.g_engine_config);
+   if (error_code != MEDIA_VISION_ERROR_NONE)
+       dlog_print(DLOG_ERROR, LOG_TAG, "error code= %d", error_code);
+   ```
+2. Decode the image file and fill the `g_source` handle with the decoded raw data.
+
+    <img src="./media/mediavision_pose_sample_sumo.png" width=300>
+
+   In the following example image `sample.jpg`, a person is shown in a squat pose, and the image is in the `<OwnDataPath>` folder.
+   The `<OwnDataPath>` refers to your own data path:
+
+   ```c
+   /* For details, see the Image Util API Reference */
+   unsigned char *dataBuffer = NULL;
+   unsigned long long bufferSize = 0;
+   unsigned long width = 0;
+   unsigned long height = 0;
+   image_util_decode_h imageDecoder = NULL;
+
+   error_code = image_util_decode_create(&imageDecoder);
+   if (error_code != IMAGE_UTIL_ERROR_NONE)
+       dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+
+   error_code = image_util_decode_set_input_path(imageDecoder, "/<OwnDataPath>/sample.jpg");
+   if (error_code != IMAGE_UTIL_ERROR_NONE)
+       dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+
+   error_code = image_util_decode_set_colorspace(imageDecoder, IMAGE_UTIL_COLORSPACE_RGB888);
+   if (error_code != IMAGE_UTIL_ERROR_NONE)
+       dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+
+   error_code = image_util_decode_set_output_buffer(imageDecoder, &dataBuffer);
+   if (error_code != IMAGE_UTIL_ERROR_NONE)
+       dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+
+   error_code = image_util_decode_run(imageDecoder, &width, &height, &bufferSize);
+   if (error_code != IMAGE_UTIL_ERROR_NONE)
+       dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+
+   error_code = image_util_decode_destroy(imageDecoder);
+   if (error_code != IMAGE_UTIL_ERROR_NONE)
+       dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+
+   /* Fill the dataBuffer to g_source */
+   error_code = mv_source_fill_by_buffer(imagedata.g_source, dataBuffer, (unsigned int)bufferSize,
+                                         (unsigned int)width, (unsigned int)height, MEDIA_VISION_COLORSPACE_RGB888);
+   if (error_code != MEDIA_VISION_ERROR_NONE)
+       dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+
+   free(dataBuffer);
+   dataBuffer = NULL;
+   ```
+3. To detect landmark of the pose from the `sample.jpg` image, create a `g_inference` media vision inference handle:
+
+   ```c
+   error_code = mv_inference_create(&imagedata.g_inference);
+   if (error_code != MEDIA_VISION_ERROR_NONE)
+       dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+   ```
+4. Configure `g_engine_config` with a body pose model data and its mapping file. Suppose that model data `data.tflite` and its mapping file `data_mapping.txt` are used as the same ones described in [Background](#background) and they are in `<OwnDataPath>`:
```

Comment:
@tyoungroy I need your opinion on the following change, kindly verify for technical correctness too. 
```suggestion
4. Configure `g_engine_config` with a body pose model data and its mapping file. Suppose that model data `data.tflite` and its mapping file `data_mapping.txt` are used as per the description in the [Background](#background) section and the files are stored in `<OwnDataPath>`:

Reply:
How about changing as follows: 
```suggestion
4. Configure `g_engine_config` with a body pose model data and its mapping file. Suppose that model data `data.tflite` and its mapping file `data_mapping.txt`, which are described in the [Background](#background) section, are applied and the files are stored in `<OwnDataPath>`:
```